Editor/Window/GameLiftPlugin.cs

```diff
@@ -3,7 +3,9 @@
 
 using System;
 using System.Collections.Generic;
+using Amazon.GameLift;
 using AmazonGameLift.Editor;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement;
```

Comment:
minor: Hmm this folder naming is strange. Consider restructuring file structure if possible since the wrappers, adapters aren't really specific to `ApiGatewayManagement`. Please clarify if I'm missing something.

Reply:
We will leave this conversation unresolved so we don't miss this. But this will be moved to a new PR(s) for Folder structure and namespaces when more things are merged in.